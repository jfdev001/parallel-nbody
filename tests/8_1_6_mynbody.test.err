Reservation number 3913415: Reserved 8 hosts for 900 seconds 
Run on 8 hosts for 960 seconds from Tue Jan 17 18:10:41 2023
: node053/0 node054/0 node055/0 node056/0 node057/0 node058/0 node063/0 node064/0 
Running N-body with 6 bodies and 1000 steps

N-body took: 0.093 seconds
Performance N-body: 0.00 GFLOPS
