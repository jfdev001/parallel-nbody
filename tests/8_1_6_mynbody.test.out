   313.023      9.014     -0.579      9.336      4.575      2.840
   875.978    766.024   -503.321   -285.733     -4.339      4.824
   512.017    214.989  -3856.220  13162.772      3.368     -2.135
   234.008     45.986   1301.622    976.752      1.619     -2.844
   842.002    746.979    447.577    215.469      0.428     -4.245
   502.013    248.995   3857.336 -12252.732      2.646     -1.027
STATUS=PASSED
